src/MCPClient/tests/test_upload_archivesspace.py

```diff
@@ -13,12 +13,11 @@
 
 
 def test_recursive_file_gen(tmpdir):
-    mydir = tmpdir.mkdir("mydir")
-    hello = mydir.join("hello.txt")
-    hello.write("hello!")
-    bye = mydir.mkdir("sub").join("bye.txt")
-    bye.write("bye!")
-    result = list(upload_archivesspace.recursive_file_gen(str(mydir)))
+    hello = tmpdir.join("mydir", "hello.txt")
+    hello.write("hello!", ensure=True)
+    bye = tmpdir.join("mydir", "sub", "bye.txt")
+    bye.write("bye!", ensure=True)
+    result = list(upload_archivesspace.recursive_file_gen(str(tmpdir / "mydir")))
```

Comment:
It looks like this approach will make sense for `test_get_files_from_dip_finds_files` as well?